.github/workflows/builder_bazel_slsa3.yml

```diff
@@ -76,6 +81,19 @@ on:
           When run on other triggers, attestations are signed and have an "intoto.sigstore" extension.
         value: ${{ jobs.slsa-run.outputs.attestations-download-name }}
 
+      provenance-download-sha256:
+        description: >
+          The sha256 digest of the attestations.
+
+          Users should verify the download against this digest to prevent tampering.
+        value: ${{ jobs.slsa-run.outputs.attestations-download-sha256 }}
+
+      binaries-download-name:
```

Comment:
do we need the corresponding `-sha256` for the output? I do't see an abvious use case, but I see other builders output it (maven, gradle, nodejs)

Reply:
> Perhaps it was just a simple way to make sure the binary wasn't altered without downloading and verifying the full provenance? (we need to do that for later publish actions etc.)

That might be it, yes.

Reply:
Added corresponding -sha256 output!